test/org/sagebionetworks/bridge/services/ScheduledActivityServiceMockTest.java

```diff
@@ -117,6 +121,11 @@ public void before() {
         SurveyResponseService surveyResponseService = mock(SurveyResponseService.class);
         when(surveyResponseService.createSurveyResponse(
             any(GuidCreatedOnVersionHolder.class), anyString(), any(List.class), anyString())).thenReturn(surveyResponse);
+
+        Subpopulation subpopulation = Subpopulation.create();
+        
+        SubpopulationService subpopService = mock(SubpopulationService.class);
+        when(subpopService.getSubpopulations(any())).thenReturn(Lists.newArrayList(subpopulation));
```

Comment:
Added, set one to have a signedOn date that's high enough it breaks things (Long.MAX_VALUE). Test at the end calls that method and verifies it's taking the lower of the two numbers.